anchors/service.go

```diff
@@ -54,7 +54,13 @@ func (s *service) GetAnchorData(anchorID AnchorID) (docRoot DocumentRoot, anchor
 	// Ignoring cancelFunc as code will block until response or timeout is triggered
 	opts, _ := s.client.GetGethCallOpts(false)
 	r, err := s.anchorRepositoryContract.GetAnchorById(opts, anchorID.BigInt())
+	if err != nil {
```

Comment:
can r be nil and not error thrown?

Reply:
let me add a check for those

Reply:
r cant be nil but blk can so added a check